<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>singleBattle</class>
 <widget class="QMainWindow" name="singleBattle">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1042</width>
    <height>705</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>1042</width>
    <height>705</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>1042</width>
    <height>705</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QPushButton" name="pushButton">
    <property name="geometry">
     <rect>
      <x>130</x>
      <y>500</y>
      <width>101</width>
      <height>41</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">color: rgb(60, 255, 197);
background-color: rgb(170, 0, 0);
font: 12pt &quot;Adobe Garamond Pro&quot;;</string>
    </property>
    <property name="text">
     <string>PushButton</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_2">
    <property name="geometry">
     <rect>
      <x>260</x>
      <y>500</y>
      <width>101</width>
      <height>41</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">color: rgb(60, 255, 197);
background-color: rgb(170, 0, 0);
font: 12pt &quot;Adobe Garamond Pro&quot;;</string>
    </property>
    <property name="text">
     <string>PushButton</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_3">
    <property name="geometry">
     <rect>
      <x>130</x>
      <y>570</y>
      <width>101</width>
      <height>41</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">color: rgb(60, 255, 197);
background-color: rgb(170, 0, 0);
font: 12pt &quot;Adobe Garamond Pro&quot;;</string>
    </property>
    <property name="text">
     <string>PushButton</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_4">
    <property name="geometry">
     <rect>
      <x>260</x>
      <y>570</y>
      <width>101</width>
      <height>41</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">color: rgb(60, 255, 197);
background-color: rgb(170, 0, 0);
font: 12pt &quot;Adobe Garamond Pro&quot;;</string>
    </property>
    <property name="text">
     <string>PushButton</string>
    </property>
   </widget>
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>180</x>
      <y>280</y>
      <width>180</width>
      <height>170</height>
     </rect>
    </property>
    <property name="text">
     <string>TextLabel</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_2">
    <property name="geometry">
     <rect>
      <x>760</x>
      <y>280</y>
      <width>180</width>
      <height>170</height>
     </rect>
    </property>
    <property name="text">
     <string>TextLabel</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_5">
    <property name="geometry">
     <rect>
      <x>880</x>
      <y>570</y>
      <width>121</width>
      <height>51</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">
background-color: rgb(170, 0, 0);
color: rgb(19, 255, 165);
font: 75 16pt &quot;Adobe Garamond Pro Bold&quot;;
</string>
    </property>
    <property name="text">
     <string>返回</string>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1042</width>
     <height>23</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>pushButton_5</sender>
   <signal>clicked()</signal>
   <receiver>singleBattle</receiver>
   <slot>close()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>936</x>
     <y>606</y>
    </hint>
    <hint type="destinationlabel">
     <x>881</x>
     <y>531</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
